### PR TITLE
Make ci pass

### PR DESCRIPTION
**Pull request by flurin-conradin**

### CODE REVIEW

**Comment by flurin-conradin** [comment]
Addresses https://github.com/juntagrico/juntagrico-custom-sub/issues/37

File: .github/workflows/juntagrico-ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: ["3.8", "3.10"]
```
Comment (pascalfree) [inline .github/workflows/juntagrico-ci.yml:19]: Base juntagrico does not officially support 3.10 yet. But will "soon". So I'm fine leaving this.

**Review by pascalfree — APPROVED** [review]
Looks legit

File: .github/workflows/juntagrico-ci.yml (hunk 1 above)
Comment (flurin-conradin) [reply .github/workflows/juntagrico-ci.yml:19]: Great! Thank you! :-)

**Pull request merged**